model/lifecycle_test.go

```diff
@@ -222,6 +222,7 @@ func TestBuildRestart(t *testing.T) {
 				Status:      evergreen.TaskDispatched,
 				Activated:   true,
 			}
+			// TODO: abortion doesn't apply for container tasks, so I think this should not be modified
```

Comment:
Container tasks can still be aborted. What does this comment mean?

Reply:
I was thinking we might have to make some edits to `RestartVersion` and, similarly, `restartTasksForBuild`, but I think this will be covered by the EVG-16305 build.

Reply:
I don't think this comment should be necessary then, since this test will presumably be updated in the EVG-16305 PR to deal with restarting/resetting tasks in a build/version. I would just remove it and assume the future PR will do the due diligence to test the change.